infra/cache/kInfraRedisCacheWrapper.php

```diff
@@ -325,7 +335,14 @@ protected function reconnect()
 		}
 
 		$connTook = microtime(true) - $connStart;
-		self::safeLog("connect took {$connTook} seconds to {$this->hostName}:{$this->port} - number of attempts: {$this->connectAttempts}");
+		if ($this->cluster)
```

Comment:
add function for getFormattedHost and check there what should be the host value, then have one print